src/front-door/azext_front_door/_params.py

```diff
@@ -276,7 +278,7 @@ def load_arguments(self, _):
         c.argument('front_door_name', frontdoor_name_type, id_part=None)
         c.argument('rules_engine_name', rules_engine_name_type, id_part=None)
         c.argument('rule_name', options_list=['--name', '-n'], help='Name of the rule')
-        c.argument('action_type', arg_group="Action", arg_type=get_enum_type(['RequestHeader', 'ResponseHeader', 'ForwardRouteOverride', 'RedirectRouteOverride']), help='Action type to apply for a rule.')
+        c.argument('action_type', arg_group="Action", arg_type=get_enum_type(['RequestHeader', 'ResponseHeader']), help='Action type to apply for a rule.')
```

Comment:
Are these two parameters no longer supported?

Reply:
It is still supported but only in 'rules-engine rule action add' command below.